src/simulator_core/entities/assets/junction.py

```diff
@@ -60,3 +60,11 @@ def create(self) -> None:
                 name=self.name,
                 in_service=self.in_service,
             )
+
+    def write_to_output(self) -> None:
+        """Placeholder to write the asset to the output.
+
+        The output list is a list of dictionaries, where each dictionary
+        represents the output of its asset for a specific timestep.
+        """
+        pass
```

Comment:
Do you only want to return none for a junction?

Reply:
DO not know yet problem is that we made the junctions and they are not in ESDL, so we cannot pass them back to database.

Reply:
Should we make an issue?

Reply:
made issue #57